allennlp/training/learning_rate_schedulers.py

```diff
@@ -226,4 +334,5 @@ def get_lr(self):
         "reduce_on_plateau": torch.optim.lr_scheduler.ReduceLROnPlateau,
         "cosine": CosineWithRestarts,
         "noam": NoamLR,
+        "slanted_triangular": SlantedTriangular
```

Comment:
Minor nit: it's slightly easier to edit this code later if you end the line with a `,`.